modules/EnsEMBL/Web/JSONServer/SpeciesSelector/Multi.pm

```diff
@@ -121,9 +124,9 @@ sub json_fetch_species {
     }
   }
 
-  if ($shown{$primary_species}) {
-    my ($chr) = split ':', $params->{"r$shown{$primary_species}"};
-    $available_species{$primary_species} = "$species_label - chromosome $chr";
+  if ($shown{$prodname}) {
```

Comment:
Could you please confirm if $shown has prod name as key or is it the url name?
Because the url for Multi is something like this `http://www.ensembl.org/Homo_sapiens/Location/Multi?db=core;r=17:63992802-64038237;r1=17:63470447-63515882:1;r2=GL873537.1:15865824-15911259:-1;s1=Pan_troglodytes;s2=Otolemur_garnettii` where the species s1 and s2 are url names. And this is what is retrieved in line 46 in this file (where %shown is initialised).  May be I have missed something but good to check.

Reply:
Ah, good point - we pass the species in as URL names. Will fix this, thanks!